_commands/basics/kill.md

```diff
@@ -0,0 +1,57 @@
+---
+---
+
+kill
+--
+
+`kill` is used to send signal to a process or to kill a process. 
+We typically use kill -SIGNAL PID, where you know the PID of the process.
+ 
+
+~~~ bash
+$ kill -9 7243
```

Comment:
I thought -9 was the signal to end the process though. At least in 485 that's what we used